neptune.yaml

```diff
@@ -52,7 +52,7 @@ parameters:
   image_channels: 3
 
 # U-Net parameters
```

Comment:
@kdziedzic66 both unets should be defined here. `pipelines_config.py` is to a large extend just a formatter of this config.